hwy/contrib/unroller/unroller_test.cc

```diff
@@ -88,6 +88,77 @@ struct ConvertUnit : UnrollerUnit<ConvertUnit<FROM_T, TO_T>, FROM_T, TO_T> {
   }
 };
 
+template <typename T>
+struct FindUnit : UnrollerUnit<FindUnit<T>, T, ptrdiff_t> {
+  static constexpr size_t UnitLanes() {
+    return HWY_MIN(HWY_MAX_LANES_D(hn::ScalableTag<T>),
+                   HWY_MAX_LANES_D(hn::ScalableTag<ptrdiff_t>));
+  }
+
+  using TT = hn::CappedTag<T, UnitLanes()>;
+  using TTSZT = hn::CappedTag<ptrdiff_t, UnitLanes()>;
```

Comment:
Strictly speaking there is no guarantee this lane type is supported. How about `MakeSigned<sizeof(ptrdiff_t)>`?

Reply:
I used `SignedFromSize` instead, but otherwise this was integrated in.

Reply:
Oops, that is what I meant 👍